.gitignore

```diff
@@ -18,6 +18,9 @@ spark-1.*-bin-hadoop*
 
 lens/lens-cli-hist.log
 
+# Zeppelin server
+zeppelin-server/local-repo
+zeppelin-server/src/main/resources/zeppelin-site.xml
```

Comment:
why this is ignored ?

Reply:
There was a discussion about this file here https://github.com/apache/zeppelin/pull/2700#discussion_r158531529.